app/js/translations/en.json

```diff
@@ -8,5 +9,34 @@
     "app.labOrdersList.URGENCY": "URGENCY",
     "app.labOrdersList.TEST_TYPE": "TEST TYPE",
     "app.labOrdersList.ORDER_DATE": "ORDER DATE",
-    "app.title": "OpenMRS Electronic Medical Record"
+    "app.labOrdersListFilters.textSearchTitle": "Search for a sample",
+    "app.labOrdersListFilters.searchDate1Label": "From: ",
```

Comment:
can you make this `app.labOrdersListFilters.searchDateFromLabel` and `app.labOrdersListFilters.searchDateFromLabel` instead of the 1 and 2

Reply:
Updated!